R/extendr-wrappers.R

```diff
@@ -1,7 +1,4 @@
 # Generated by extendr: Do not edit by hand
-
-# nolint start
-
```

Comment:
Could you use the dev version rextendr?
(Installs from GitHub or R-universe)

Reply:
Sure, no problem: 007e675a755cc56d1d57ea70518ca1063818d43d